incubator/zalenium/README.md

```diff
@@ -0,0 +1,96 @@
+| `hub.sauceLabsEnabled` | Enable SauceLabs | false |
+| `hub.browserStackEnabled` | Enable BrowserStack | false |
+| `hub.testingBotEnabled` | Enable TestingBot | false |
+| `hub.videoRecordingEnabled` | Enable video recording | true |
+| `hub.cpuRequest` | CPU requested for browser pods.  The hub passes this value to the k8s API | 200m |
+| `hub.cpuLimit` | CPU limit for browser pods.  The hub passes this value to the k8s API | 400m |
+| `hub.memRequest` | Memory requested for browser pods.  The hub passes this value to the k8s API | 200Mi |
+| `hub.screenWidth` | Screen resolution to use | 1440 |
+| `hub.screenHeight` | Screen resolution to use | 900 |
+| `hub.timeZone` | Time zone | UTC |
+| `hub.seleniumImageName` | The Selenium grid image | `elgalu/selenium` |
+| `hub.maxTestSessions` | The number of tests to run on each grid container before killing it and starting a new one | 1 |
+| `hub.debugEnabled` | 	Enables LogLevel.FINE | false |
+| `hub.keepOnlyFailedTests` | Keeps only failed tests on the dashboard (you need to send a cookie with the test result) | false |
+| `hub.sendAnonymousUsageInfo` | Allows sending anonymous usage info | true |
+| `hub.sauceUserName` | Username to log into saucelabs. Use if hub.sauceLabsEnabled set to true | blank |
+| `hub.sauceAccessKey` | Access key to log into saucelabs. Use if hub.sauceLabsEnabled set to true | blank |
+| `hub.browserStackUser` | Credentials for browserstack. Use if hub.browserStackEnabled set to true | blank |
+| `hub.browserStackKey` | Credentials for browserstack. Use if hub.browserStackEnabled set to true | blank |
+| `hub.testingBotKey` | Credentials for testingbot. Use if hub.testingBotEnabled set to true | blank | 
+| `hub.testingBotSecret` | Credentials for testingbot. Use if hub.testingBotEnabled set to true | blank |
+| `ingress.enabled` | Set ingress.enabled to true if you want to create an ingress entry for zalenium | false |
+| `ingress.hostname` | Set ingress.hostname to which host you want | `#` |
+| `services.type` | Set services.type | `ClusterIP` |
+| `mountStorage.videos.enabled` | Whether to use a videos mount volume. Set mountStorage.videos.enabled to true | true |
+| `mountStorage.data.enabled` | Whether to use a data mount volume. Set mountStorage.data.enabled to true | true |
+| `mountStorage.videos.pvcs.enabled` | Whether to use a persistent volume claim for storage. Set mountStorage.videos.pvcs.enabled to true | false |
+| `mountStorage.data.pvcs.enabled` | Whether to use a persistent volume claim for storage. Set mountStorage.data.pvcs.enabled to true | false |
+
+Specify each parameter using the `--set key0=value0,key1=value1,key..n=value..n` argument to `helm install`. For example,
+
+```console
+$ helm install --name example-name \
+  --set rbac.create=false \
+    teracy-incubator/zelenium
+```
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart.
+
+## How to run
```

Comment:
I still don't see any first impression here, please demonstrate it here, not a link.